### PR TITLE
[Fixed] Position and color of back to top button

### PR DESCRIPTION
**Pull request by JoyBhalla**
Changed the color from purple to blue,changed the position of the button a little,let me know what can be improved.
References #89

### CODE REVIEW

**Comment by AniVerma17** [comment]
Provide a preview link please.

File: css/layout.css [hunk 1]
```diff
@@ -1018,20 +1018,20 @@ section.communityBox .section-header{
...
      cursor: pointer;
-     background-color: #45006e;
+     background-color: #1a8cff;
      padding: 15px;
      box-shadow: 0 0 5px 0 #ccccff;
 	 transition: all 0.2s linear;
 }
 #up:hover,#up:active{
-     background-color: #620090;
+     background-color: #1a8cff;
```
Comment (AniVerma17) [inline css/layout.css]: Make this color of a bit different shade (preferably a darker one) to give it a visible hover/touch effect.

**Comment by JoyBhalla** [comment]
screenshot from 2018-04-02 07-43-50
can't set it up in gh-pages rn,but i have lowered the button a little and so it doesn't look obstructive.

File: css/layout.css (hunk 1 above)
Comment (JoyBhalla) [reply css/layout.css]: Do you mean a darker blue or a darker color in general ?

File: index.html [hunk 2]
```diff
@@ -877,9 +877,16 @@ <h3>Contact us</h3>
...
     <div id="up">
-     <span class="glyphicon glyphicon-chevron-up"></span>
+          <span class="glyphicon glyphicon-chevron-up"></span>
+
     </div>
 
+
+<!--Contact Us-->
+
+<div class="react-card">
```
Comment (thisisayush) [inline index.html]: What's the role of this div added?

Comment (JoyBhalla) [reply index.html]: Sorry galat push kar diya I was trying something out,delete karna bhul gaya ghar jaake dekhta hun

Comment (thisisayush) [reply index.html]: Cool. Rest Seems OK :+1:

**Comment by JoyBhalla** [comment]
I don't have my laptop with me right now,I'll do it around 9.

File: css/layout.css (hunk 1 above)
Comment (AniVerma17) [reply css/layout.css]: A darker blue.

Comment (JoyBhalla) [reply css/layout.css]: screenshot from 2018-04-02 07-43-50
a little darker,subtle difference tho

**Comment by thisisayush** [comment]
@JoyBhalla Please squash the commits to one. Refer Contribution Guidelines in ```README``` for more info.

**Comment by JoyBhalla** [comment]
Squashed commits into one,do i have to push changes again after squashing ?

**Comment by AniVerma17** [comment]
`Squashed commits into one,do i have to push changes again after squashing ?
Yes

**Comment by JoyBhalla** [comment]
Done

**Comment by AniVerma17** [comment]
I still see 2 commits, they aren't squashed yet.

**Comment by JoyBhalla** [comment]
Okay,let me look into it.

**Comment by thisisayush** [comment]
@JoyBhalla Yeah. You need to force push after squashing the commits, add `-f` option to `push origin master` i.e. `push origin master -f`

**Comment by JoyBhalla** [comment]
Changes have been made let me know if you've checked it out.

**Pull request merged**